packages/client/modules/userDashboard/components/OrgTeams/ArchiveTeamModal.tsx

```diff
@@ -0,0 +1,69 @@
+import {Close} from '@mui/icons-material'
+import React from 'react'
+import DialogTitle from '../../../../components/DialogTitle'
+import FlatButton from '../../../../components/FlatButton'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import useMutationProps from '../../../../hooks/useMutationProps'
+import useRouter from '../../../../hooks/useRouter'
+import ArchiveTeamMutation from '../../../../mutations/ArchiveTeamMutation'
+
+type Props = {
+  teamId: string
+  teamName: string
+  teamLeadName: string | null
+  closeModal: () => void
+}
+
+const ArchiveTeamModal = (props: Props) => {
+  const {teamId, teamName, teamLeadName, closeModal} = props
+  const {onCompleted, onError, submitMutation, submitting} = useMutationProps()
+  const atmosphere = useAtmosphere()
+  const {history} = useRouter()
+
+  const handleClose = () => {
+    closeModal()
+  }
+
+  const onSubmit = async () => {
+    if (submitting) return
+    submitMutation()
+    await ArchiveTeamMutation(atmosphere, {teamId}, {history, onError, onCompleted})
+    closeModal()
+  }
+
+  const confirmationText = (
+    <span>
+      {`Are you sure you want to delete ${teamLeadName ? `${teamLeadName}'s team` : 'your team'} "${teamName}"?`}
+      <br />
+      <span className='mt-2 block text-sm text-slate-600'>
+        <b>Note</b>: This can’t be undone.
```

Comment:
+1 The note is gray and does not stand out much. I would rephrase it slightly, remove the "Note: " part and make it bold instead:
Are you sure you want to permanently delete your team Foo?
**This can't be undone.**